src/commands/sites/sites-list.ts

```diff
@@ -21,17 +21,21 @@ export const sitesList = async (options: OptionValues, command: BaseCommand) =>
   }
 
   if (sites && sites.length !== 0) {
-    // @ts-expect-error TS(7006) FIXME: Parameter 'site' implicitly has an 'any' type.
     const logSites = sites.map((site) => {
-      const siteInfo = {
+      const siteInfo: {
+        id: string
+        name: string
+        ssl_url: string
+        account_name: string
+        repo_url?: string
+      } = {
```

Comment:
Instead of defining a new type here you could do something like
```suggestion
      const siteInfo: Pick<SiteInfo, 'id' | 'name' | etc...> = {
```